### PR TITLE
Transaction view update

### PR DESCRIPTION
**Pull request by KillariDev**
- uses grids instead of texts to display things
- use blockies for unknown tokens
- approving 0 is now positive event (appears green)
- quite a bit refactoring for types and for how coins.tsx modules work (now they don't determine what is positive and what negative anymore)

image
image
image

### CODE REVIEW

File: extension/app/css/interceptor.css [hunk 1]
```diff
@@ -450,3 +454,30 @@ svg.spinner > circle {
...
+
+.log-cell {
+	text-overflow: ellipsis;
+	overflow: hidden;
+	white-space: nowrap;
+	display: flex;
+}
+.ellipsis {
+	text-overflow: ellipsis;
+    overflow: hidden;
```
Comment (MicahZoltu) [inline extension/app/css/interceptor.css:482]: Weird indentation.

Comment (KillariDev) [reply extension/app/css/interceptor.css]: fixed

File: extension/app/ts/components/subcomponents/icons.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+export function ArrowIcon(param: { color: string }) {
+	return <>
+		&nbsp;
```
Comment (MicahZoltu) [inline extension/app/ts/components/subcomponents/icons.tsx:3]: Feels like we should be using styling here rather than `&nbsp;`?  e.g., `padding`

Comment (MicahZoltu) [reply extension/app/ts/components/subcomponents/icons.tsx:3]: Same below and I think I saw another one potentially above.

**Review by MicahZoltu — APPROVED** [review]
This PR is huge, so approving.

**Pull request merged**